pyproject.toml

```
@@ -21,6 +21,7 @@ gunicorn = "~=20.0.4"
 sentry-sdk = "~=0.19"
 markdown = "~=3.3.4"
 python-frontmatter = "~=1.0"
+django-prometheus = "^2.1.0"
```

```suggestion
django-prometheus = "~=2.1"
```
This should be changed to follow the format of the other pinned dependencies.